pkg/unittest/validators/snapshot_validator.go

```diff
@@ -41,7 +41,7 @@ func (v MatchSnapshotValidator) Validate(context *ValidateContext) (bool, []stri
 		return false, splitInfof(errorFormat, -1, err.Error())
 	}
 
-	validateSuccess := false
+	validateSuccess := true
```

Comment:
validateSuccess := len(manifests) == 0

Set the validationSucces, based on the length.
This way when a manifest is found, we still have the same validation.